src/plugins/intel_cpu/src/config.cpp

```diff
@@ -255,6 +255,34 @@ void Config::readDebugCapsProperties() {
     if (envVarValue = readEnv("OV_CPU_BLOB_DUMP_NODE_NAME"))
         blobDumpFilters[BY_NAME] = envVarValue;
 
+    auto isFuseeFilter = [](const std::string& filterStr) {
+        return filterStr.front() == '<';
+    }; // Otherwise consider as Fuser filter
+
+    auto isFuserFilter = [](const std::string& filterStr) {
+        return filterStr.front() == '>';
+    }; // Otherwise consider as Fuser filter
```

Comment:
Could you please align the names Fuser->Target, Fusee->Fused? Here and  below.

Reply:
Corrected